cmake/InstallChemfiles.cmake

```diff
@@ -34,8 +34,8 @@ endif()
 
 ExternalProject_Add(
     CHEMFILES
-    GIT_REPOSITORY https://github.com/frodofine/chemfiles.git
-    GIT_TAG merge_for_dubs
+    GIT_REPOSITORY https://github.com/chemfiles/chemfiles.git
+    GIT_TAG master
```

Comment:
Side note, you may want to track specific commit instead of master, to ensure someone can buil 0.3.1 in 5 years if they want to reproduce a result obtained with lemon 😃

Reply:
That's a great idea!